x-pack/plugins/fleet/server/services/epm/archive/extract.ts

```diff
@@ -19,14 +21,22 @@ export async function untarBuffer(
 ): Promise<unknown> {
   const deflatedStream = bufferToStream(buffer);
   // use tar.list vs .extract to avoid writing to disk
-  const inflateStream = tar.list().on('entry', (entry: tar.FileStat) => {
-    const path = entry.header.path || '';
+  const inflateStream = tar.list().on('entry', (entry) => {
+    const path = entry.path || '';
```

Comment:
Accessing the `path` via the `header` works but isn't supported according to the types. Instead it seems that you're supposed to access it directly via the root `path` property where it's copied to in the constructor: 

https://github.com/npm/node-tar/blob/26a496e5fa74eeaa0c3539511560fc181ef56557/lib/read-entry.js#L50